packages/emails/templates/organizer-scheduled-email.ts

```diff
@@ -14,13 +15,15 @@ export default class OrganizerScheduledEmail extends BaseEmail {
   calEvent: CalendarEvent;
   t: TFunction;
   newSeat?: boolean;
+  teamMember?: Person;
 
-  constructor(calEvent: CalendarEvent, newSeat?: boolean) {
+  constructor(calEvent: CalendarEvent, newSeat?: boolean, teamMember?: Person) {
```

Comment:
As there are 2 optional parameters now, it is highly recommended to use an object

```suggestion
  constructor({
  calEvent, 
  newSeat, 
  teamMember
  }:{
  calEvent: CalendarEvent, newSeat?: boolean, teamMember?: Person
  }) {
```

Reply:
This form is easier for others to understand.

Reply:
https://github.com/calcom/cal.com/pull/7207/commits/8f3f17b2997ea67f1417be64e15ca4b71a736917